XenAdmin/TabPages/ManageUpdatesPage.Designer.cs

```diff
@@ -203,10 +210,42 @@ private void InitializeComponent()
             this.toolStripButtonExportAll,
             this.toolStripLabelFiltersOnOff,
             this.toolStripSplitButtonDismiss,
-            this.toolStripButtonRestoreDismissed});
+            this.toolStripButtonRestoreDismissed,
+            this.toolStripSeparator});
             this.toolStrip1.Name = "toolStrip1";
             this.toolStrip1.TabStop = true;
             // 
+            // toolStripSplitButtonView
+            // 
+            this.toolStripSplitButtonView.DisplayStyle = System.Windows.Forms.ToolStripItemDisplayStyle.Text;
+            this.toolStripSplitButtonView.DropDownItems.AddRange(new System.Windows.Forms.ToolStripItem[] {
+            this.byUpdateToolStripMenuItem,
+            this.byHostToolStripMenuItem});
+            resources.ApplyResources(this.toolStripSplitButtonView, "toolStripSplitButtonView");
+            this.toolStripSplitButtonView.Name = "toolStripSplitButtonView";
+            this.toolStripSplitButtonView.DropDownItemClicked += new System.Windows.Forms.ToolStripItemClickedEventHandler(this.toolStripSplitButtonView_DropDownItemClicked);
+            // 
+            // byUpdateToolStripMenuItem
+            // 
+            this.byUpdateToolStripMenuItem.Checked = true;
+            this.byUpdateToolStripMenuItem.CheckState = System.Windows.Forms.CheckState.Checked;
+            this.byUpdateToolStripMenuItem.Image = global::XenAdmin.Properties.Resources.notif_updates_16;
+            this.byUpdateToolStripMenuItem.Name = "byUpdateToolStripMenuItem";
+            resources.ApplyResources(this.byUpdateToolStripMenuItem, "byUpdateToolStripMenuItem");
+            this.byUpdateToolStripMenuItem.Click += new System.EventHandler(this.byUpdateToolStripMenuItem_Click);
+            // 
+            // byHostToolStripMenuItem
+            // 
+            this.byHostToolStripMenuItem.Image = global::XenAdmin.Properties.Resources._000_TreeConnected_h32bit_16;

```

Comment:
The images look a bit stretched and blurry, you probably need to try the other scaling options in their properties.